HomeTask2/HomeTask2/CheckingAndTranslateSymbols.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace HomeTask2
+{
+    class CheckingAndTranslateSymbols
+    {
+        // this function check each input symbol
+        // return true, if all symbol is letter
+        // else return false
+        public bool CheckingSymbols(string[] args)
+        {
+            char[] argToChar;
+            foreach(string arg in args)
+            {
+                argToChar = arg.ToCharArray();
+                foreach(char symbol in argToChar)
```

Comment:
Why do you check only for letters? what about over symbols? you have to determine is it a russian or translit texts